File: apollo/src/main/java/io/muun/apollo/domain/errors/TooManyWrongVerificationCodesError.java
package io.muun.apollo.domain.errors;


import io.muun.apollo.external.UserFacingErrorMessages;

public class TooManyWrongVerificationCodesError extends UserFacingError {

    public TooManyWrongVerificationCodesError() {
        super(UserFacingErrorMessages.INSTANCE.tooManyWrongVerificationCodes());
    }
}

File: android/apolloui/src/main/java/io/muun/apollo/domain/errors/p2p/TooManyWrongVerificationCodesError.kt
package io.muun.apollo.domain.errors.p2p


import io.muun.apollo.data.external.UserFacingErrorMessages
import io.muun.apollo.domain.errors.ErrorClassification
import io.muun.apollo.domain.errors.UserFacingError

class TooManyWrongVerificationCodesError : UserFacingError(
    UserFacingErrorMessages.INSTANCE.tooManyWrongVerificationCodes()
) {
    override val classification = ErrorClassification.EXPECTED
}
